swift/Tests/Blockchains/BItcoinTests.swift

```diff
@@ -80,4 +80,45 @@ class BitcoinTransactionSignerTests: XCTestCase {
             "00000000" // nLockTime
         )
     }
+
+    func testSignP2SH_P2WPKH() {
```

Comment:
Strange filename

Reply:
I think git mv can change case in filename

Reply:
yeah, I noticed as well